ClientLibrary/Snippets/Microsoft.TeamServices.Samples.Client/WorkItemTracking/WorkItemsIconSample.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.TeamFoundation.WorkItemTracking.WebApi;
+using Microsoft.TeamFoundation.WorkItemTracking.WebApi.Models;
+using Microsoft.VisualStudio.Services.WebApi;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.TeamServices.Samples.Client.WorkItemTracking
+{
+    /// <summary>
+    /// Client samples for getting work item icons in Team Services and Team Foundation Server.
+    /// </summary>
+    [ClientSample(WitConstants.WorkItemTrackingWebConstants.RestAreaName, "workItemIcons")]
```

Comment:
There should be a constant for "workItemIcons" so you don't have to hard-code the string

Reply:
Yep, added that to VSTS it will be in next drop